lib/i18n/tasks/missing_keys.rb

```diff
@@ -154,5 +149,32 @@ def collapse_same_key_in_locales!(forest)
       end
       forest
     end
+
+    private
+
+    def plural_keys_for_locale(locale)
+      configuration = load_rails_i18n_pluralization!(locale)
+      if configuration[locale.to_sym].nil?
+        alternate_locale = alternate_locale_from(locale)
+        return Set.new if configuration[alternate_locale].nil?
+        return set_from_rails_i18n_pluralization(configuration, alternate_locale)
+      end
+      set_from_rails_i18n_pluralization(configuration, locale)
+    rescue SystemCallError, IOError
+      Set.new
+    end
+
+    def alternate_locale_from(locale)
+      re = /(\w{2})-*(\w{2})*/
+      match = locale.match(re)
+      language_code = match[1]
+      country_code = match[2]
+      country_code.upcase
```

Comment:
This doesn't actually modify the `country_code` but the test didn't fail

Reply:
You're right, I think this solved my problem because it returned an empty set for `en-GB`.

Reply:
Updated the code and tests, thanks for catching that.